packages/perseus-editor/src/components/widget-editor.tsx

```diff
@@ -30,6 +30,7 @@ type WidgetEditorProps = {
     ) => unknown;
     onRemove: () => unknown;
     apiOptions: APIOptions;
+    widgetIsOpen?: boolean;
```

Comment:
So with the terms I propose, what do you think of simply having this prop be `expanded?: boolean`?

Reply:
Are you suggesting `expanded` be the prop from the `editor-page` down to the `widget-editor` or just for the `widget-editor` portion of the code?

Reply:
I think I might be splitting hairs here.  I was thinking this component would use `expanded` but as we move up that means the other components should change to be consistent. Changing prop names is relatively easy so why don't we just leave things as you have them and land the PR?  

If we see that over time it's confusing or we find a better set of terms, we can change it.

Reply:
Okay, thanks!